composer.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "benjaminhu/tsheets-api-phplib",
```

Can we change this to:
```suggestion
    "name": "tsheets/tsheets-api-phplib",
```